javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,25 @@
+// ####### Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+
+// Write some code here
+function removeName(removalName) {
```

Comment:
This is very good that you have created a function, but I can see that you don't use the parameter that is defined in the function. You could for instance make your function reusable by passing two parameters , I will leave the rest for you to figure out.

Reply:
you are right, my intention putting parameter is using for function but instead i hardcoded it 🤦🏽‍♂️